GWFish/modules/fishermatrix.py

```diff
@@ -234,7 +243,7 @@ def compute_detector_fisher(
     signal_parameter_values: Union[pd.DataFrame, dict[str, float]],
     fisher_parameters: Optional[list[str]] = None,
     waveform_model: str = wf.DEFAULT_WAVEFORM_MODEL,
-    waveform_class: type(wf.Waveform) = wf.LALFD_Waveform,
+    waveform_class = wf.LALFD_Waveform,
```

Comment:
Why is the type hint removed here?
Eventually we should get rid of the `waveform_class` parameter altogether so it doesn't matter that much, but I'm curious: was it giving you trouble?

Reply:
No trouble at all. I wanted to see "compute_detector_fisher" and "compute_network_errors" with the same input syntax.